server/test/unit/server/webservices/views/test_decorators.py

```diff
@@ -1,14 +1,14 @@
 """
-This module contains tests for the pulp.server.webservices.controllers.decorators module.
+This module contains tests for the pulp.server.webservices.views.decorators module.
 """
 import mock
 
 from .... import base
 from pulp.server.exceptions import PulpCodedAuthenticationException
-from pulp.server.webservices.controllers import decorators
+from pulp.server.webservices.views import decorators
 
 
-class TestAuthenticationMethods(base.PulpWebserviceTests):
+class TestAuthenticationMethods(base.PulpServerTests):
```

Comment:
PulpWebserviceTests was responsible for controllers tests. https://github.com/pulp/pulp/blob/master/server/test/unit/base.py#L87